internal/pkg/service/buffer/config/config_test.go

```diff
@@ -77,7 +77,7 @@ sink:
             volumeAssignment:
                 # Volumes count simultaneously utilized per sink. Validation rules: required,min=1,max=100
                 count: 1
-                # List of preferred volume types, start with the most preferred. Validation rules: min=1
+                # List of preferred volume types, start with the most preferred. Validation rules: required,min=1
```

Comment:
In the last commit, a slighty updated some validarion rules, so that they are not confusing.

`min=1` is sufficient validation, but I prefer to mark all required values with the `required` rule.